packages/create-atomic-component-project/index.js

```diff
@@ -63,9 +63,16 @@ const main = () => {
       destPath: 'package.json',
       transform: (content) => {
         const packageJson = JSON.parse(content);
-        for (const script of packageJson.scripts) {
-          script.name = script.name.substring(1);
+
+        if (packageJson.scripts) {
```

Comment:
These were failing on execution, since scripts is not an array but an object with key-values